locales/en/text-curation.json

```diff
@@ -18,6 +18,7 @@
   },
   "dataset_description": "Dataset Description",
   "date_columns": "Date Columns",
+  "Download": "Download",
```

Comment:
@prakhar-s please follow key naming consistency, it has to be lowercase, without space (use underscore instead)

Reply:
done